playbooks/libvirt/openshift-cluster/tasks/configure_libvirt_storage_pool.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create libvirt storage directory for openshift
+  file:
+    dest: "{{ libvirt_storage_pool_path }}"
+    state: directory
+
+- acl:
+    default: yes
+    entity: kvm
+    etype: group
+    name: "{{ libvirt_storage_pool_path }}"
+    permissions: rwx
+    state: present
+
+- name: Test if libvirt storage pool for openshift already exists
+  command: "virsh -c {{ libvirt_uri }} pool-info {{ libvirt_storage_pool }}"
+  register: pool_info_result
+  changed_when: False
+  failed_when: "pool_info_result.rc != 0 and 'error: Storage pool not found:' not in pool_info_result.stderr"
```

Comment:
This test fails when a locale is set.
Unfortunately, replacing
`command: "virsh -c {{ libvirt_uri }} pool-info {{ libvirt_storage_pool }}"`
by:
`shell: "LANG=POSIX virsh -c {{ libvirt_uri }} pool-info {{ libvirt_storage_pool }}"`
does **not** help.
See https://github.com/detiber/openshift-ansible/pull/20 for a solution proposal